statsd-exporter/include/mappings-high-cardinality.yml

```diff
@@ -0,0 +1,75 @@
+# This config undoes the changes done in https://github.com/astronomer/ap-vendor/pull/37/files
+# so that we can continue to support customers who need these metrics still.
+
```

Comment:
I think we should get rid of this text. This PR is suitable for giving us context about why this file exists.
```suggestion
```